src/renderers/shared/stack/reconciler/instantiateReactComponent.js

```diff
@@ -106,6 +106,11 @@ function instantiateReactComponent(node) {
       // representations. I.e. ART. Once those are updated to use the string
       // representation, we can drop this code path.
       instance = new element.type(element);
+
+      // We renamed this. Allow the old name for compat. :(
+      if (!instance.getHostNode) {
+        instance.getHostNode = instance.getNativeNode;
+      }
```

Comment:
Since we didn't ship ReactART 15 yet… we could just make it depend on 15.1 and not support 15.0.